### DISCUSSION TITLE
Any reason why we now need to target net6.0-windows10.0.17763 instead of net6.0-windows7.0 for ReactiveUI.WPF?

### DISCUSSION THREAD

**Discussion by yexcoffier** [reactiveui/ReactiveUI#3517] [Q&A] [answered]
Hi, my projects currently target net6.0-windows (which targets net6.0-windows7.0). But since this commit : https://github.com/reactiveui/ReactiveUI/commit/19bc7954171a9fe5c352617908b29111ffd1ed79 (after 18.4.1)
we now have to indicate net6.0-windows10.0.17763 instead (net6.0-windows10.0 doesn't work either).
So I updated my projects and it works, it's not a huge deal, but is there a specific reason why?

It's still much more practical when editing a csproj to use net6.0-windows / net7.0-windows instead of remembering the Windows minimal accepted version.

**Comment by ChrisPulman** [comment, marked as answer]
Windows 7, which is end of life is not supported by some of the dependencies of the ReactiveUI libraries. We try to target as many supported targets as possible.

https://support.microsoft.com/en-us/windows/windows-7-support-ended-on-january-14-2020-b75d4580-2cc7-895a-2c9c-1466d9a53962
https://github.com/dotnet/core/blob/main/release-notes/7.0/supported-os.md#out-of-support-os-versions

**Comment by yexcoffier** [comment]
Ok, thank you for your answer!

**Comment by sgf** [comment]
in short: 
If you want to support (windows7,windows8/8.1,windows 10 and buildnumber<17763)  obviously updating ReactiveUI is not a good idea.

**Reply by glennawatson** [reply]
Yep we don't support it. If you need to target windows 7 feel free to use a older version of rxui
